src/pages/jobs.md

```diff
@@ -40,13 +72,154 @@ all remote, distributed team.
 More information about the company we're trying to build can be found in our
 [public handbook](https://github.com/shorebirdtech/handbook)
 
-# Founding Engineer
+---
+
+# Systems Engineer
 
 We write the tools others will use to build Flutter apps, including the compiler
-used to build them and runtime used to execute them. We’re looking for someone
-who enjoys working down at the "lower" levels of our system including the
-forks of the Flutter and Dart runtime and compiler.
+used to build them and runtime used to execute them. The ideal candidate is
+comfortable in the "lower" levels of our stack, including C++ programming on
+mobile.
 
 As one of our first engineers, you will wear many hats and work across many
 systems over time. You will work with a diverse and distributed team of
 exceptional engineers to build the future of development for _any screen_.
+
+## Responsibilities
+
+You will be responsible for many of the "lowest levels” of the product, working
+on the C++/C/Rust code that powers Dart VM and compiler, Flutter’s engine and
+Shorebird’s updater. This will include working on performance at these layers
+and working with the rest of the team to maintain our forks of millions of lines
+of C++ code.
+
+You will be responsible for many design decisions within the product, and will
+be expected to be able to make tradeoffs between different approaches. You will
+be expected to be able to work within a distributed team of engineers to build a
+product that is easy to use, performant, secure and delightful to developers.
+That starts of course by building something we ourselves want to use.
+
+## Location
+
+We're all-remote, current team members are located in California, Illinois and
+New York (GMT-8 through GMT-5). Will consider GMT-10 through GMT-3. We
+communicate exclusively through Discord (both in public and private) including
+often leaving video calls open in the background while we work independently.
+
+We get together 3-4x per year as a company, all flying to some US city.
+
+## Compensation
+
+180-210k USD salary, 1.5% equity, and
+[benefits](https://handbook.shorebird.dev/benefits/).
+
+## Requirements
```

Comment:
This restates a bunch of stuff in the previous sections. Maybe we could roll the responsibilities and location sections into this?

Reply:
Addressed